[esx]/es_extended/client/functions.lua

```diff
@@ -22,7 +22,7 @@ ESX.Scaleform.Utils = {}
 ESX.Streaming = {}
 
 function ESX.SetTimeout(msec, cb)
-    table.insert(Core.TimeoutCallbacks, {
+    ESX.TableInsert(Core.TimeoutCallbacks, {
```

Comment:
ESX.TableInsert?

Reply:
oh didnt see that u made a function for it 🤔

Reply:
Yes 🙂